test/fixtures/smart_proxies.yml

```diff
@@ -31,13 +26,7 @@ four:
 bmc:
   name: BMC proxy
   url: http://else.where:45673
-  features: bmc
 
 logs:
   name: A proxy with Logs feature
-  url: http://else.where:45674
-  features:
-    - dhcp
-    - dns
-    - puppet
-    - logs
+  url: http://else.where:45674
```

Comment:
Grab a chance and get rid of this completely? ;-) Don't judge me, just trying...

Reply:
It's also good to use the `example.com` in tests/examples. So `else.where.example.com`

Reply:
@lzap i'd rather not refactor all the existing tests :)
just 'else.where' seems referenced 29 times in the foreman tests, it seems out of the scope of this pr.